tests/integration/pip_parse/.bazelrc

```diff
@@ -0,0 +1,11 @@
+# Bazel configuration flags
+
+build --enable_runfiles
+startup --windows_enable_symlinks
+
+# https://docs.bazel.build/versions/main/best-practices.html#using-the-bazelrc-file
+try-import %workspace%/user.bazelrc
+
+# The requirements.bzl entry_point functions aren't supported under bzlmod.
+# They are replaced by py_console_script_binary, which already has tests
+build --noexperimental_enable_bzlmod
```

Comment:
Lets leave bzlmod enabled by default, to match the default config

Reply:
```suggestion
# The requirements.bzl entry_point functions aren't supported under bzlmod.
# They are replaced by py_console_script_binary, which already has tests
build --enable_bzlmod
```